bitnami/elasticsearch/templates/coordinating-statefulset.yaml

```
@@ -44,6 +44,7 @@ spec:
       {{- if .Values.coordinating.schedulerName }}
       schedulerName: {{ .Values.coordinating.schedulerName }}
       {{- end }}
+      priorityClassName: {{ .Values.coordinating.priorityClassName | quote }}
```

This should be wrapped in an if statement to check if `priorityClassName` is set. Like this: 

```suggestion
      {{- if .Values.coordinating.priorityClassName }}
      priorityClassName: {{ .Values.coordinating.priorityClassName | quote }}
      {{- end }}
```

The same applies to the other statefulsets.